tron/config/static_config.py

```diff
@@ -0,0 +1,23 @@
+from functools import partial
+
+import staticconf
+from staticconf import config
+
+FILENAME = "/nail/srv/configs/tron.yaml"
+NAMESPACE = "tron"
+
+
+def load_yaml_file():
```

Comment:
```suggestion
def load_yaml_file() -> staticconf.YamlConfiguration:
```

Reply:
`` load_yaml_file()`` doesn't have a return value. Should I still add a type hint to the return value of this function? 
or 
```
def load_yaml_file() -> None
```

Reply:
whoops, my brain made up a return: you're right, this should be `-> None`